tests/e2e/default-mobile/reports/bulk-delete.wdio-spec.js

```diff
@@ -0,0 +1,106 @@
+const utils = require('../../../utils');
+const commonElements = require('../../../page-objects/default/common/common.wdio.page');
+const reportsPage = require('../../../page-objects/default-mobile/reports/reports.wdio.page');
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+
+describe('Bulk delete reports', () => {
+  const docs = [
+    {
+      fields: {
+        lmp_date: 'Feb 3, 2016'
+      },
+      form: 'P',
+      type: 'data_record',
+      content_type: 'xml',
+      reported_date: 1462333250374,
+      contact: {
+        _id: '3305E3D0-2970-7B0E-AB97-C3239CD22D32'
+      },
+      from: '+555',
+      hidden_fields: []
+    },
+    {
+      fields: {
+        lmp_date: 'Feb 15, 2016'
+      },
+      form: 'P',
+      type: 'data_record',
+      content_type: 'xml',
+      reported_date: 1462338250374,
+      contact: {
+        _id: '3305E3D0-2970-7B0E-AB97-C3239CD22D32'
+      },
+      from: '+555',
+      hidden_fields: []
+    },
+    {
+      fields: {
+        ok: 'Yes!'
+      },
+      form: 'V',
+      type: 'data_record',
+      content_type: 'xml',
+      reported_date: 1462538250374,
+      contact: {
+        _id: '3305E3D0-2970-7B0E-AB97-C3239CD22D32'
+      },
+      from: '+555',
+      hidden_fields: []
+    },
+    {
+      name: 'Sharon',
+      phone: '+555',
+      type: 'person',
+      reported_date: 1462538250374,
+      _id: '3305E3D0-2970-7B0E-AB97-C3239CD22D32'
+    }
+  ];
+
+  const savedUuids = [];
+  before(async () => {
+    await loginPage.cookieLogin();
```

Comment:
When we use the `cookieLogin()` method we use an online user with the `admin` role, and we have a pending ticket to update the automated tests that we have to be executed with an offline user, [this one](https://github.com/medic/cht-core/issues/5704).

Could you please create an offline user and execute the complete test using that user?

Reply:
Sure! Done